sct-commons/src/test/java/org/lfenergy/compas/sct/commons/util/UtilsTest.java

```diff
@@ -482,6 +483,30 @@ void copySclElement_should_throwException() {
                 .hasMessage("org.lfenergy.compas.sct.commons.dto.FCDAInfo is not known to this context");
     }
 
+    @Test
+    void copySclElement_should_succeed_when_syncRead() throws ExecutionException, InterruptedException {
+        // Given
+        TLN tln = new TLN();
+        tln.setLnType("T1");
+        tln.getLnClass().add(TLLN0Enum.LLN_0.value());
+        ExecutorService service = Executors.newFixedThreadPool(2);
+        Callable<TLN> copySclElementTlnCallable = new Callable<TLN>() {
+            @Override
+            public TLN call() {
+                return copySclElement(tln, TLN.class);
+            }
+        };
```

Comment:
You can declare a Callable with a lambda : 
```suggestion
        Callable<TLN> copySclElementTlnCallable = () -> copySclElement(tln, TLN.class);
```